src/components/AccountView.jsx

```diff
@@ -15,7 +15,7 @@ const AccountView = (props) => {
     return <p>Loading...</p>
   }
 
-  const attributes = !instance.data ? instance.data.attributes : {}
+  const attributes = instance.data.attributes
```

Comment:
What did you want to achieve ? Set default attributes ? Did you consider this trick :
```javascript
const attributes = instance.data && instance.data.attributes || {}
```